ui/components/privacy-view.jsx

```diff
@@ -0,0 +1,145 @@
+         According to GSA’s Common Questions About DAP, “none of the federal government data tracked as part of the Data Analytics Program will be shared with or available to Google’s corporate advertising partners.”
+
+        A limited number of authorized individuals will have user accounts that will allow them to log in to the Google Analytics dashboard and view or run reports regarding visits to the OMB Policy Library and the other web metrics available from the DAP.
+
+        Visitors who choose to disable this web measurement tool will still have full access to the OMB Policy Library.
+        While the details vary from browser to browser, most modern browsers can be set up to accept, reject, or request user intervention when a site asks to set a cookie.
+
+        You can view web metrics information at https://analytics.usa.gov//.
+        {/*eslint-enable */}
+      </p>
+
+      <h4>Information you choose to give the OMB</h4>
+      <p className="max-width-3">
+        {/*eslint-disable */}
+        Users of this website may send the OMB feedback or report an issue by sending an email to ofcio@omb.eop.gov.
+        If you choose to send us your personally identifiable information, we will only use that information to respond to your message.
+         We only share the information you give us with another government agency if your question relates to that agency, or as otherwise required by law.
+         The OMB Policy Library never collects information or creates individual profiles for the purposes of advertising, marketing, or any other commercial purpose.
+          We do not create or store lists of email addresses or other information to deliver email alerts, newsletters, or otherwise send marketing messages or contact individuals.
+           When you contact us, any personally identifiable information you provide is voluntary.
+           Please do not include sensitive personally identifiable information or other sensitive information in the content of your email.
+           {/*eslint-enable */}
+      </p>
+
+      <h4>Children and privacy on the OMB policy library</h4>
+      <p className="max-width-3">
+        {/*eslint-disable */}
+        We believe in the importance of protecting the privacy of children online.
+         The Children’s Online Privacy Protection Act (COPPA) governs information gathered online from or about children under the age of 13.
+         This site is not intended to solicit or collection information of any kind from children under age 13.
+         If you believe that we have received information from a child under age 13, please contact us at ofcio@omb.eop.gov.
+         {/*eslint-enable */}
+      </p>
+
+
+      <h4>Security</h4>
+      <p className="max-width-3">
+        {/*eslint-disable */}
+        This website was built using Sites, a government-hosted service designed to enable the rapid deployment of government websites in a secure and readily accessible environment.
```

Comment:
We're not using Sites, correct? Happy to rephrase this if someone can update me on the relevant things we are using.